Python/di_sensors/easy_mutex.py

```diff
@@ -0,0 +1,23 @@
+'''
+MUTEX HANDLING
+'''
+import I2C_mutex
+mutex = I2C_mutex.Mutex(debug = False)
+overall_mutex = mutex.overall_mutex()
```

Comment:
Are we sure `overall_mutex` method is going to do it's job all the time? 
I'm seeing posts on the internet saying `os.path.isfile` could throw a `FileNotFoundError` exception. 

We wouldn't want to use mutexes if the user has specified not to.

Reply:
Havent seen any issues yet

Reply:
Researching more onto this, I don't see any comments on exceptions on `os.path.isfile` in the official documentation, which theoretically and most likely, this means the only 2 outcomes are of either getting a `False` or a `True`.

Still, I don't see why posts on `FileNotFoundError` would appear. And if you say you haven't found any issues, then it must be this way I think.